teuthology/provision/pelagos.py

```diff
@@ -71,18 +71,23 @@ def __init__(self, name, os_type, os_version=""):
             self.os_name = os_type
         self.log = log.getChild(self.name)
 
-    def create(self):
+    def create(self, init_only=False):
         """
         Initiate deployment via REST requests and wait until completion
-
+        :param init_only: if set, function exits just after reprovisioning
```

Comment:
Can we rename the `init_only` to `wait` and defaults to `True`, so if you want to wait till the creation finished use:
```
create(wait=False)
```
if want to return immediately after creation request issued.